README.md

```diff
@@ -1,3 +1,55 @@
-# SwornDisk v2
+# MlsDisk
 
-SwornDisk v2 adopts on a new layered design and implementation.
+## Introduction
+
+MlsDisk is a **multilayered log-structured secure virtual disk for TEEs**, which has the following key features:
+
+* **Transparent protection.** As a virtual disk, MlsDisk can _transparently_ protect any file system (e.g., Ext4) that is stacked upon it and runs inside a TEE from a strong adversary outside the Trusted Execution Environment (TEE).
+
+* **Strong security.** MlsDisk promises six security properties: _confidentiality_, _integrity_, _freshness_, _consistency_, _atomicity_, and _irreversibility_. For more information, see [the security goal](#security-goal) below.
+
+* **High performance.** Thanks to its unique log-structured, amplification-minimized design, MlsDisk can deliver an excellent I/O performance that is close to the theoretically optimal level.
+
+## Security Goal
+
+MlsDisk targets a typical setting of TEE usage, where applications are ported into the TEE with no (or few) modifications thanks to a TEE-aware runtime. For enclave TEEs (e.g., Intel SGX), one popular choice for such a runtime is library OSes (e.g., [Occlum](https://github.com/occlum/occlum)). For VM TEEs (e.g., AMD SEV), one can choose off-the-shelf OS kernels like Linux.
+
+![The threat model of MlsDisk.](./docs/figures/mlsdisk-threat-model.png)
```

Comment:
Here are suggestions for improve the figure.

## Issue 1

In the figure, the green square is labeled as 

> Secure Block Device
> (secure block device)

Apparently, the two lines are redundant. I suggest use the following labels for the square.

>  Secure Block Device
> (e.g., MlsDisk)

This would make a figure that depicts the threat model of secure block devices for TEEs in general.

If you want to emphasize that the threat model is for MldSisk in particular, then this purpose can be served better by the following labels.

> MlsDisk
> (secure block device)

## Issue 2

Remove the address mapping attacks. This type of attack is only mentioned in on specific version of the paper. And I don't think the attack is included in the paper any more.

## Issue 3

Think twice before adding binary files into version control. 

> # Should I Use Git LFS?
>
> The reason you should use Git LFS or an alternative if you manage large or binary files is that Git is decentralized. This means every developer has the full change history on their computer. Changes in large binary files cause Git repositories to grow by the size of that file every time the file is changed (and that change is committed). That means it will take a long time to get the files. And if you do, it will be difficult to version and merge the binaries.
>
> So, every time the files grow, the Git repository grows. This causes slowdowns when Git users need to retrieve and clone a repository.

It won't be the end of the day to have a few figures, PDFs, or other binary files included in the Git history. But keep in mind: every time you update a binary file, even when you only change one character in the binary, the entire binary is considered changed by Git and the size of the entire binary is added to that of the Git history.

Let's get back to the case of this PR. Will the two figures be updated in the future? I think the answer is probably yes. This is especially true for the performance results. So having them included in the Git history is of a concern.

So I suggest handle the two two figures in different ways.
1. For the threat model figure, convert it to the svg format. This should reduce the file size from hundreds of KB to tens of KB. This does not change the fact of including binary files, but it does make version controling the file more tolerable.
2. For the benchmark figure, simply replace it with a few Markdown tables. We can't report the full results in the README anyway; it will be done in the paper. So here for the README, showing simplified data in the table format is good enough.